### PR TITLE
Use single server for both ws and http

### PR DESCRIPTION
**Pull request by ali-bahjati**

### CODE REVIEW

File: third_party/pyth/price-service/src/rest.ts [hunk 1]
```diff
@@ -126,7 +127,7 @@ export class RestAPI {
       }
     );
     endpoints.push(
-      "latest_vaas?ids[]=<price_feed_id>&ids[]=<price_feed_id_2>&.."
+      "api/latest_vaas?ids[]=<price_feed_id>&ids[]=<price_feed_id_2>&.."
```
Comment (thmzlt) [inline third_party/pyth/price-service/src/rest.ts:130]: These new endpoints with the `api/` prefix, it looks like the consumers on the other side will need to be aware of the change. Just wanted to make sure that this is being taken in account.

**Comment by thmzlt** [comment]
I'm not familiar with the p2w codebase, but this looks good. I'll leave it to another reviewer to give explicit approval.

**Review by jayantk — APPROVED** [review]
nice

File: third_party/pyth/price-service/src/rest.ts (hunk 1 above)
Comment (ali-bahjati) [reply third_party/pyth/price-service/src/rest.ts:130]: Yes, the next PR fixes them.

**Pull request merged**